client/src/Components/Comment/Comment.js

```diff
@@ -1,15 +1,23 @@
+/* eslint-disable no-unused-vars */
 import React, { useState } from 'react';
 import 'bootstrap/dist/css/bootstrap.css';
 import { Link } from 'react-router-dom';
-import useCommentsFetch from '../../Hooks/useCommentsFetch';
-import Reactions from '../Reactions/Reactions.js';
 import { MAX_COMMENT_DEPTH } from '../../constants.js';
+import Reactions from '../Reactions/Reactions.js';
+import api from '../../api.js';
 
 const Comment = (props) => {
 	const [upvotes, setUpvotes] = useState(props.upvotes);
 	const [status, setStatus] = useState(props.status);
-	const [loadComments, setLoadComments] = useState(false);
-	const comments = useCommentsFetch(props.id);
+	const [showChildren, setShowChildren] = useState('unloaded');
```

Comment:
I'm going crazy when I don't know what the custom states here could be, aren't you? 😅
How many states here could be?
Might enum be useful here? Or even boolean (since you can use it combining with the `comments.length`)?